Marlin/Marlin_main.cpp

```diff
@@ -5503,10 +5511,13 @@ inline void gcode_M226() {
    * M300: Play beep sound S<frequency Hz> P<duration ms>
    */
   inline void gcode_M300() {
-    uint16_t beepS = code_seen('S') ? code_value_short() : 110;
-    uint32_t beepP = code_seen('P') ? code_value_long() : 1000;
-    if (beepP > 5000) beepP = 5000; // limit to 5 seconds
-    buzz(beepP, beepS);
+    uint16_t const frequency = code_seen('S') ? code_value_short() : 110;
```

Comment:
`Const` here is a compiler hint and a best practice, when you have a dynamic var which you know will not change after being assigned, it can [or should] be declared const. Also it's worth noting an optimization that can be done on the function's argument list, consider the following function prototype:

```cpp
bool enqueue(T const &item) ();
```

I'm not only declaring that my function code will not change the `item` variable as forcing the compiler not to duplicate the original variable and just pass me the address of the original `item`. Of course this is valid only for arguments that will change by my function's code.